docs/conditions.md

```diff
@@ -1052,7 +1052,7 @@ CONDITION_STAND_STATE</p></td>
 <td>CONDITION_QUESTSTATE</td>
 <td>47</td>
 <td>Quest ID - see <a href="http://www.azerothcore.org/wiki/quest_template#id">quest_template.id</a></td>
-<td>state_mask</td>
+<td>state_mask (1 = not taken, 2 = completed, 8 = in progress, 32 = failed, 64 = rewarded)</td>
```

Comment:
tbh I would prefer it being in a bullet pin to make it easier for the eye.
Like this:

state_mask:
* Not Taken = 1
* Completed = 2
* In Progress = 8
* Failed = 32
* Rewarded = 64

Good catch though! Didn't even know this myself :D

Reply:
also I can look into adding this to Keira :D
And out of curiousity where in the core did you find this, will be good for me to learn too :P ?

Reply:
i don't remember, maybe in something like QuestMgr.h

to be fair I grepped the src for "state_mask" because no info was given anywhere, not even on TC


for the bullet I don't know, I just added it quickly, it's not even working at the moment because we don't have the condition so i don't care about its design lol

Reply:
Fix the design and maybe just maybe I can look into how it is supposed to work :P

Reply:
you know you can edit what i've done

I can't do it at the moment because when I try to edit it always says "Unable to load this preview, sorry." so if I'd need to push first, then see if it's okay, but i'm sure it won't work fine the first time

Reply:
I just didnt want to intrude :P